src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java

```diff
@@ -0,0 +1,206 @@
+package com.wikia.webdriver.testcases.seotests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * @ownership Seo
+ */
+public class HtmlTitleTests extends NewTestTemplate {
+
+  Credentials credentials = Configuration.getCredentials();
+
+  @DataProvider
+  private Object[][] dataHtmlTitleTest001() {
+    return new Object[][]{
+        // original title
+        {
+            "sktest123",
```

Comment:
@justnpT what it brings for us?
If you may say that it would be easy to change in the future I can say that every simple editor supports `s/old_string/new_string/g` feature and of course modern IDE.
I know that DRY principle is perfect but it works for code, not a raw data.
I would rather concentrate on separating tests and data completely to different files. In such a way we can use the data for more than one test from different files and if we would like to change something in the data file it will be stored in one place.